framework/include/tinyalsa/limits.h

```diff
@@ -43,6 +43,18 @@
 ** DAMAGE.
 */
 
+/**
+* @defgroup TinyAlsa TinyAlsa
+* @ingroup TinyAlsa
+* @brief Provides APIs for Audio Framework
+* @{
+ */
+
+/**
+ * @file limits.h
+ * @brief All macros, structures and functions that make up the PCM interface.
```

Comment:
All of brief are same. Do they have same fucntionality? Then why do we split files?